tests/install_upgrade_operators/crypto_policy/utils.py

```
@@ -261,9 +272,144 @@ def update_apiserver_crypto_policy(
+        bool: True if the service accepted the TLS connection.
+    """
+    command = compose_openssl_command(
+        service_spec=service.instance.spec,
+        version=tls_version,
+        extra_arguments="| grep 'Protocol version:'",
+    )
+    try:
+        output = ExecCommandOnPod(utility_pods=utility_pods, node=node).exec(command=command, ignore_rc=True)
+    except ExecOnPodError:
+        service_name = service.instance.metadata.name
+        LOGGER.warning(f"Service {service_name} is unreachable during TLS {tls_version} check, treating as rejected")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)